Dream-Home-PEABOT/FEE_DreamHome
# Deploy

### What does this PR do?

### This is the deployed branch, any deployment changes should be pushed here

* Adds changes to support heroku deployment
* Removes unused imports and variables
* Removes whitespace and comments

### Where should the reviewer start?

* Start by checking for any unused data

### How should this be manually tested?

* Check [deployed link](https://dream-home-cap.herokuapp.com/) and see that the layout is displaying

### Any background context you want to provide?

* Api can be connected locally through `localhost:3000` but otherwise will not be connected

### What are the relevant tickets?

closes #68
